amm/src/main/scala/ammonite/interp/ClassLoaders.scala

```diff
@@ -118,7 +118,7 @@ class SpecialClassLoader(parent: ClassLoader, parentSignature: Seq[(Path, Long)]
   }
 
   private def jarSignature(url: URL) = {
-    val path = Path(url.getFile, root)
```

Comment:
Actually `url.getFile` doesn't converts urls back to OS specific path.
http://stackoverflow.com/questions/6164448/convert-url-to-normal-windows-filename-java

Also `url.getFile` is now deprecated